src/components/try-hcp-callout/components/product-icon-heading/index.tsx

```diff
@@ -0,0 +1,37 @@
+import InlineSvg from '@hashicorp/react-inline-svg'
+import hcpLogo from '!!raw-loader!@hashicorp/mktg-logos/product/hcp/primary/black.svg'
+import ProductIcon from 'components/product-icon'
+import { ProductSlug } from 'types/products'
+import s from './product-icon-heading.module.css'
+
+export interface ProductIconHeadingProps {
+	productSlug: Exclude<ProductSlug, 'sentinel'>
+	headingText: string
+}
+
+/**
+ * A card heading component, for use in HCP callout cards.
+ *
+ * Note that when "hcp" is provided as the product slug, we render
+ * the full HCP logo, and we render the provided headingText as visually hidden.
+ */
+export function ProductIconHeading({
+	productSlug,
+	headingText,
+}: ProductIconHeadingProps) {
+	if (productSlug === 'hcp') {
+		return (
+			<div className={s.rootHcpLogo}>
+				<InlineSvg src={hcpLogo} />
+				<span className="g-screen-reader-only">{headingText}</span>
+			</div>
```

Comment:
🚧 The screen reader experience for this variant is very close! It just needs one small adjustment.

Both the `InlineSvg`'s underlying `div` and the visually hidden `span` can be navigated to with the arrow keys with VoiceOver. This means something that is visually presented as one element is non-visually presented as two elements, and that's a mismatched experience. Here's a demonstration (no audio, need to download CleanShot again for better recordings!!):

https://user-images.githubusercontent.com/43934258/202585526-7687c444-efae-48fb-b2fe-85d14164a4e3.mov

To fix this mismatched experience and always present this as one element, I believe we need the code to look like this:

```suggestion
			<>
				<span id="auto-generated-id" className="g-screen-reader-only">
					{headingText}
				</span>
				<div aria-labelledby="auto-generated-id" role="presentation">
					<InlineSvg aria-hidden className={s.rootHcpLogo} src={hcpLogo} />
				</div>
			</>
```

List of the changes:

- Replaces wrapper `div` with a fragment (`InlineSvg` renders a `<div>`)
- Moves `s.rootHcpLogo` class to `InlineSvg` (gets applied to it's rendered outer `div`)
- Moves visually hidden span to be first (not sure if this is needed, but it feels right to place it before the element that references it)
- Adds an `id` to the visually hidden span (We should auto generate this with `useId` from `@react-aria/utils` because multiple HCP variants of this component can `technically` be rendered on the same page)
- Sets `aria-hidden` on the `InlineSvg`, and wrap it with a `div` with `aria-labelledby` that references the visually hidden span and `role="presentation"`. Explanations:
  - When `aria-labelledby` is set on `InlineSvg`, it's still presented as two elements because the rendered `<div>` is the labelled element, and it has an `svg` with a `title` inside of it
  - Without `role="presentation"`, it's still presented as two elements and the first one is presented as "HashiCorp Cloud Platform, empty group".

Here is a demonstration of how these changes sound with VoiceOver in Chrome:

https://user-images.githubusercontent.com/43934258/202598199-f9d10fc6-fb7d-4e80-a796-758226089166.mov

I realize this a long comment for a small code change... 😆 So I'm happy to sync on this via Slack / quick Zoom if that'd be helpful! 😊

Reply:
💖 Thanks for diving in here, I'll work through this list of changes! 💯

Reply:
✅ okey dokes, this should be fixed up, I hope! Thanks for the direct suggestion here - I applied that and then brought in `useId` 👍 I think that covers the list of changes!

```ts
const uniqueId = useId()

if (productSlug === 'hcp') {
	return (
		<>
			<span id={uniqueId} className="g-screen-reader-only">
				{headingText}
			</span>
			<div aria-labelledby={uniqueId} role="presentation">
				<InlineSvg aria-hidden className={s.rootHcpLogo} src={hcpLogo} />
			</div>
		</>
	)
}
```

Reply:
Looks great!! Skip the first few seconds of this recording (I _really_ need to download Cleanshot again 🤣) and that's the output I got testing the preview URL 👍🏻 

https://user-images.githubusercontent.com/43934258/202780960-598e1ca0-01a8-46ac-ad1a-fcf1238467cf.mov